### PR TITLE
Dynamize dashboard

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: ApiBundle/Controller/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{
+    /**
+     * @param Request $request
+     *
+     * @Config\Route("", name="open_orchestra_api_dashboard")
+     * @Config\Method({"GET"})
+     *
+     * @return FacadeInterface
+     */
+    public function listWidgetsAction(Request $request)
```
Comment (itkg-alavieille) [inline ApiBundle/Controller/DashboardController.php]: can you add this action in api controller test

File: BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php [hunk 2]
```diff
@@ -67,6 +67,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('authorize_status_change.yml');
         $loader->load('authorize_edition.yml');
         $loader->load('restore_entity.yml');
+        $loader->load('dashboard.yml');
```
Comment (SOLenG) [inline BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php]: I think you forgot merge the site's configuration with the yaml

Comment (itkg-ngilain) [inline BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php]: totally right ^^'
I work on that

Comment (itkg-ngilain) [inline BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php]: @SOLenG : after a new reflexion it's not a so good idea.
This file only presents a parameter (an array) describing the widgets used on the dashboard (not the widget availbales). So if the integrator wants to use other widgets for his project, he just has to override that array in his project conf. By merging it with our, we'll force our widgets to be present, so merge is not a good thing for that param.

**Comment by itkg-ngilain** [comment]
@itkg-alavieille : PR updated

**Comment by itkg-alavieille** [comment]
:+1:

File: BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php (hunk 2 above)
Comment (nicolasThal) [inline BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php]: In each case, the configuration that the integrator has set is not analyzed, and in each case the dashboard will contain only the few one we set.

I don't know if the merge is the best solution, but I think that a solution inbetween could be found :)

Comment (itkg-ngilain) [inline BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php]: @nicolasThal : If the integrator alters his app/config/parameters.yml, his settings replace the Orchestra default ones. Do you think we should provide another way to override that parameter ?

**Comment by itkg-ngilain** [comment]
@SOLenG , @nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**